frontend/src/components/KBar/KBarWrapper.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <KBarProvider :actions="globalActions" :options="{ disabled }">
+    <KBarPortal>
+      <KBarPositioner class="mask">
+        <KBarAnimator class="container">
+          <KBarSearch class="searchbox" />
+          <KBarHelper />
+          <RenderResults />
+          <KBarFooter />
+        </KBarAnimator>
+      </KBarPositioner>
+    </KBarPortal>
+
+    <slot />
+  </KBarProvider>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed } from "vue";
+import {
+  KBarProvider,
+  KBarPortal,
+  KBarPositioner,
+  KBarAnimator,
+  KBarSearch,
+  createAction,
+} from "@bytebase/vue-kbar";
+import { useStore } from "vuex";
+import { useRouter } from "vue-router";
+import RenderResults from "./RenderResults.vue";
+import KBarHelper from "./KBarHelper.vue";
+import KBarFooter from "./KBarFooter.vue";
+
+export default defineComponent({
+  name: "KBarWrapper",
+  components: {
+    KBarProvider,
+    KBarPortal,
+    KBarPositioner,
+    KBarAnimator,
+    KBarSearch,
+    RenderResults,
+    KBarHelper,
+    KBarFooter,
+  },
+  setup() {
+    const store = useStore();
+    const router = useRouter();
+
+    const disabled = computed(() => {
+      const currentUser = store.getters["auth/currentUser"]();
+      // totally disable kbar when not logged in
+      return !currentUser || currentUser.id < 0;
+    });
+
+    const globalActions = [
+      createAction({
+        id: "home",
+        name: "Home",
+        shortcut: ["g", "h"],
```

Comment:
Just double, do we need to launch the kbar and type "gh" to visit home? If that's the case, I am actually leaning towards to just use "h". Because "gh" could be used later by the direct shortcut key (similar to the linear behavior)

Reply:
Shortcuts are global. Just hit "gh", "gp", "gd" ... for a quick navigation without opening the command bar.
P.S. shortcuts are not available if you are focusing on an textbox or textarea.

Reply:
ACK